docs/schema/clientwebpartpropertytype-simpletype-apphostwebfeatures.md

```diff
@@ -18,11 +19,9 @@ ms.assetid: aed2ce76-7e2d-84df-e80b-496d5b4bd3d8
 
 ## Type information
 
-|   |   |
-|---|---|
-| **Base Type**  | xs:string |
-| **Namespace**  | `http://schemas.microsoft.com/sharepoint/` |
-| **Schema file**  | apphostwebfeatures.xsd |
+| Base Type | Namespace | Schema File |
```

Comment:
Please turn this into a 2 table column with suggested headers of "Name/Value" or "Information/Location" or whatever you feel is most appropriate.